rest/taskana-rest-spring-example/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java

```diff
@@ -126,7 +130,7 @@ public void testGetSecondPageSortedByKey() {
 
     @Test
     public void testCreateClassification() throws IOException {
-        String newClassification = "{\"classificationId\":\"\",\"category\":\"MANUAL\",\"domain\":\"DOMAIN_A\",\"key\":\"NEW_CLASS\",\"name\":\"new classification\",\"type\":\"TASK\"}";
+        String newClassification = "{\"classificationId\":null,\"category\":\"MANUAL\",\"domain\":\"DOMAIN_A\",\"key\":\"NEW_CLASS\",\"name\":\"new classification\",\"type\":\"TASK\"}";
```

Comment:
We should also have a test returning a HTTP 400 if ID is set.

Reply:
ok